### ISSUE TITLE
Axes chop as fast as your hand

### ISSUE THREAD

**Issue by 10203** [SWUTM/Trails_and_Tales_Forge#3]
In the 1.18.2 version of the mod from Modrinth the mod prevents axes from working correctly on any wooden object. The axes don't mine any faster than your hand, this extends to modded axes and modded wood types and objects.

**Comment by AyoubSen** [comment]
yep I'm not sure if it is planned to be fixed or not.. also leaves are mined as a block instead of a leave.. so its taking as long to remove
